airbrakes/data_handling/data_processor.py

```diff
@@ -310,7 +279,7 @@ def _calculate_vertical_velocity(self) -> npt.NDArray[np.float64]:
         vertical_accelerations = np.array(
             [
                 deadband(vertical_acceleration - GRAVITY, ACCELERATION_NOISE_THRESHOLD)
-                for vertical_acceleration in self._rotated_accelerations[self._gravity_axis_index]
+                for vertical_acceleration in self._rotated_accelerations[2]
```

Comment:
if we are sure you can always index it at 2, how about we make it so self._rotated_accelerations is just a numpy array (or list I guess) that just has that 3rd column. No need to do the list comprehensions and math for the other 2 columns

Reply:
Will do